test/test_format.py

```diff
@@ -9,9 +9,8 @@
     Field,
     MsgFormat,
     SrvFormat,
-    ActionFormat,
 )
-from roswire.ros1 import ROS1PackageDatabase, ROS1FormatDatabase
+from roswire.ros1 import ROS1ActionFormat, ROS1PackageDatabase, ROS1FormatDatabase
```

Comment:
prefer relative import

Reply:
I think because this is a test case, we should do the `roswire.`, which is consistent with the other test cases.